py2pack/__init__.py

```diff
@@ -336,10 +341,51 @@ def _normalize_license(data):
 def _prepare_template_env(template_dir):
     # setup jinja2 environment with custom filters
     env = jinja2.Environment(loader=jinja2.FileSystemLoader(template_dir))
+
+    def _rpm_format_requires(req):
```

Comment:
nitpick: This function does complex require formatting, it could be great to have some comment with examples of what it does, input -> output so it will be easier to understand in the future.